src/js/components/SpaceModal.js

```diff
@@ -42,14 +44,17 @@ export default function SpaceModal() {
 const SpaceModalContents = ({value, onChange}) => {
   return (
     <div className="space-modal-contents">
-      <Input
-        autoFocus
-        label="Space Name"
-        value={value}
-        onChange={(e) => {
-          onChange(e.target.value)
-        }}
-      />
+      <InputField>
+        <InputLabel>New Name:</InputLabel>
+        <TextInput
+          autoFocus
+          label="Space Name"
+          value={value}
+          onChange={(e) => {
+            onChange(e.target.value)
+          }}
+        />
+      </InputField>
```

Comment:
<img width="701" alt="Screen Shot 2020-06-22 at 12 49 37 PM" src="https://user-images.githubusercontent.com/3460638/85329630-bcd6cd80-b487-11ea-8f44-9a6b9b974bc6.png">

Reply:
I think that `label="Space Name"` prop/attribute in the `<TextInput />` will not be of use anymore.

Reply:
Also, I'm aware that putting colons after labels is helpful for older screen readers to parse forms but from a design perspective I find I prefer these labels only with the emphasis we are already giving them (boldness, placement, followed by a clearly editable box) and no colon. Just one person's opinion though :)